src/models/fhir/required.ts

```diff
@@ -15,7 +15,9 @@ export const hasRequiredFields = (type: Type, parsedBundle: ParsedBundle) => {
   });
 
   if (errors) {
-    throw new DocumentInvalidError(
+    throw new CodedError(
+      "INVALID_DOCUMENT",
+      `Submitted HealthCert is invalid`,
       `the following required fields in fhirBundle are missing: ${JSON.stringify(
         errors
       )}. For more info, refer to the mapping table here: https://github.com/Open-Attestation/schemata/pull/38`
```

Comment:
Let's move the details to the message (as details are not sent in the API response)

Reply:
Done! Modified to use the error message which was defined in the test file 

Modified code: https://github.com/Notarise-gov-sg/api-notarise-healthcerts/blob/ec70f97b5148f51977bf14158bcc4ca373cfed48/src/models/fhir/required.ts#L20

Test file message example: https://github.com/Notarise-gov-sg/api-notarise-healthcerts/blob/ec70f97b5148f51977bf14158bcc4ca373cfed48/src/models/fhir/required.test.ts#L40